code/game/gamemodes/miniantags/pulsedemon/pulsedemon_event.dm

```diff
@@ -29,14 +32,14 @@
 	var/list/spawn_centers = list()
 	for(var/datum/regional_powernet/P in SSmachines.powernets)
 		for(var/obj/structure/cable/C in P.cables)
-			if(!is_station_level(C.z) || P.available_power <= 0)
+			if(!is_station_level(C.z) || P.available_power < DEMON_REQUIRED_POWER)
```

Comment:
think its worth adding a log error to this just so admins know why pulsy failed to spawn?